runtime/src/main/java/org/corfudb/runtime/collections/TxnContextShim.java

```diff
@@ -18,22 +28,19 @@
  * Created by hisundar on 2020-09-16
  */
 @Slf4j
-public class TxnContextShim implements AutoCloseable {
-    /**
-     * Internal CorfuStore's txnContext
-     */
-    private TxnContext txnContext;
+public class TxnContextShim extends TxnContext {
```

Comment:
will this not implement auto-closeable anymore?

Reply:
Thanks! added the interface